builder/azure/arm/step_deploy_template_test.go

```diff
@@ -114,7 +117,7 @@ func TestStepDeployTemplateDeleteImageShouldFailWithInvalidImage(t *testing.T) {
 
 func TestStepDeployTemplateCleanupShouldDeleteManagedOSImageInExistingResourceGroup(t *testing.T) {
 	var deleteDiskCounter = 0
-	var testSubject = createTestStepDeployTemplateDeleteOSImage(&deleteDiskCounter)
+	var testSubject = createTestStepDeployTemplateDeleteOSImage(&deleteDiskCounter, true)
```

Comment:
Same thing here it is not clear what the boolean is for. But here you can create a variable initialized to true.